VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -14,12 +13,24 @@
         "version": "1"
     },
 
-    "VirtoCommerce": {
-        "ActivationUrl": "https://virtocommerce.com/admin/api/licenses/activate/",
-        "LicenseFilePath": "App_Data\\VirtoCommerce.lic",
-        "LicensePublicKeyPath": "App_Data\\public.key"
-    },
+  "VirtoCommerce": {
+    "ActivationUrl": "https://virtocommerce.com/admin/api/licenses/activate/",
+    "LicenseFilePath": "App_Data\\VirtoCommerce.lic",
+    "LicensePublicKeyPath": "App_Data\\public.key",
+    "UploadUrl": "~/App_Data/Uploads/"
+  },
+    "Assets": {
+        "Provider": "FileSystem",
+        "FileSystem": {
+            "rootPath": "~/App_Data/Assets",
```

Comment:
Use tilda less relative path (see LicensePublicKeyPath and  LicenseFilePath settings as example)